mods.go

```diff
@@ -352,9 +352,33 @@ func (m *Mods) startCompletionCmd(content string) tea.Cmd {
 				ccfg.APIType = openai.APITypeAzureAD
 			}
 		default:
-			key, err := m.ensureKey(api, "OPENAI_API_KEY", "https://platform.openai.com/account/api-keys")
-			if err != nil {
-				return err
+			var key string
+			if cfg.API == "copilot" {
```

Comment:
these should probably be below, where there is a switch case on cfg.API

Reply:
Are you sure there's a switch on `cfg.API`? I can't find it.

Reply:
Yeah, I tried to make it work inside the `mod.API` switch statement but rolled back the changes because I wanted to keep as much of the OpenAI logic as possible, and only change the `env` and `http` endpoint values.

Reply:
Yeah, I tried to make it work inside the mod.API switch statement but rolled back the changes because I wanted to keep as much of the OpenAI logic as possible, and only change the env and http endpoint values.

Reply:
I don't think that's a good idea, as the way you did it will also use the openai model from the configuration, which I don't think is correct.

Reply:
Yes, it will only work if you set it to `gpt-4o` or one its supported variants. But there's no need to be too specific. Using `gpt-4o` is enough.